### PR TITLE
Update distribution matcher in spec file

### PR DESCRIPTION
**Pull request by schaefi**
Updates the distribution matchers as needed for the open buildservice

### CODE REVIEW

**Comment by schaefi** [comment]
I'm missing SLES13

I'd appreciate a review from the obs team

File: package/python-kiwi-spec-template [hunk 1]
```diff
@@ -19,27 +19,41 @@
...
 # SLE12:
 %if 0%{?suse_version} == 1315 && !0%{?is_opensuse}
 %define distro suse-SLES12
 %endif
-# Leap 42.1:
-%if 0%{?suse_version} == 1315 && 0%{?is_opensuse}
+
+# SLE13:
+%if 0%{?sle_version} == 130000
+%define distro suse-SLES13
```
Comment (DimStar77) [inline package/python-kiwi-spec-template]: Careful here - this is likely also to catch Leap 43.0

Comment (schaefi) [inline package/python-kiwi-spec-template]: ok how about it now ?

File: package/python-kiwi-spec-template [hunk 2]
```diff
@@ -30,7 +30,7 @@
 %endif
 
 # SLE13:
-%if 0%{?sle_version} == 130000
+%if 0%{?sle_version} == 130000 && !0%{?is_opensuse}
```
Comment (DimStar77) [inline package/python-kiwi-spec-template]: LGTM

**Pull request merged**